conan/api/subapi/upload.py

```diff
@@ -13,13 +11,11 @@ def __init__(self, conan_api):
         self.conan_api = conan_api
 
     @api_method
-    def get_bundle(self, expression, package_query=None, only_recipe=False):
-        ref_pattern = SelectPattern(expression)
```

Comment:
This has been extracted to the command, to align with all other List related commands.